test/platform/test-platform-identify.ts

```diff
@@ -0,0 +1,91 @@
+/// <reference types="@google/local-home-sdk" />
+import test from 'ava';
+import {
+  MockLocalHomePlatform,
+  ERROR_UNDEFINED_IDENTIFYHANDLER,
+  ERROR_UNDEFINED_APP,
+  ERROR_LISTEN_NOT_CALLED,
+  ERROR_UNDEFINED_VERIFICATIONID,
+} from '../../src/platform/mock-local-home-platform';
+import {AppStub} from '../../src/platform/smart-home-app';
+import {injectSmarthomeStubs} from '../../src';
+
+test.before(t => {
+  injectSmarthomeStubs();
+});
+
+test('only-trigger-identify-when-platform-initialized', async t => {
```

Comment:
Would have broken into further tests but singleton pattern makes this difficult.  Any suggestions?

Reply:
If you're having issues with the singleton, you may want to come up with a way to mock the instance or have a way to do a reset. Being able to initialize to a clean state will be useful.

Reply:
Yeah good point - this is a testing tool anyways.  I'll add a method to reset the state

Reply:
Added optional parameter to reset state in singleton getter.  Split up tests.